src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -42,8 +42,9 @@ class AccessDecisionManager implements AccessDecisionManagerInterface
      */
     public function __construct($voters = [], $strategy = self::STRATEGY_AFFIRMATIVE, $allowIfAllAbstainDecisions = false, $allowIfEqualGrantedDeniedDecisions = true)
     {
+        $strategy = (string) $strategy;
         $strategyMethod = 'decide'.ucfirst($strategy);
-        if (!\is_callable([$this, $strategyMethod])) {
+        if (!\is_callable([$this, $strategyMethod]) || 0 === \strlen($strategy)) {
```

Comment:
i would add an extra if, validating the strategy explicitly. Like done here https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Cookie.php#L129 (same topic to me).